ros2topic/ros2topic/verb/info.py

```diff
@@ -25,6 +51,10 @@ def add_arguments(self, parser, cli_name):
         arg = parser.add_argument(
             'topic_name',
             help="Name of the ROS topic to get info (e.g. '/chatter')")
+        parser.add_argument(
+            '--verbose', '-v', action='store_true',
+            help='Prints detailed information like the Node Name, Node Namespace, Topic Type, '
```

Comment:
@jaisontj @mm318 consider:
```suggestion
            help='Prints detailed information like the node name, node namespace, topic type, '
```
instead

Reply:
Done.